src/components/reviews/reviewTile.jsx

```diff
@@ -22,13 +55,51 @@ class ReviewTile extends React.Component {
     });
   }
 
+  handleHelpful() {
```

Comment:
Possibly think about pulling this out into a separate file to declutter this file.